apps/prairielearn/src/lib/db-types.ts

```diff
@@ -341,3 +341,53 @@ export const AssessmentsFormatForQuestionSchema = z.array(
     color: z.string().nullable(),
   }),
 );
+
+export const AssessmentQuestionSchema = z.object({
+  advance_score_perc: z.number().nullable(),
+  alternative_group_id: IdSchema.nullable(),
+  assessment_id: IdSchema,
+  average_average_submission_score: z.number().nullable(),
+  average_first_submission_score: z.number().nullable(),
+  average_last_submission_score: z.number().nullable(),
+  average_max_submission_score: z.number().nullable(),
+  average_number_submissions: z.number().nullable(),
+  average_submission_score_hist: z.array(z.number()).nullable(),
+  average_submission_score_variance: z.number().nullable(),
+  deleted_at: DateFromISOString.nullable(),
+  discrimination: z.number().nullable(),
+  effective_advance_score_perc: z.number().nullable().default(0),
```

Comment:
We don't express database-level defaults in the Zod schema. These schemas should never change what we get back from the DB; they should only enforce that they have the expected shape/types.

Reply:
Done in 3f76666.